src/DropdownMenu/DropdownMenu.js

```diff
@@ -96,9 +97,13 @@ export default class DropdownMenu extends React.Component {
   };
 
   templateMenu = ({ isOpen, options, height, onMenuClick }) => {
-    return (
-      isOpen ? <Menu {...{ options }} maxHeight={height} onClick={onMenuClick} /> : null
-    );
+    return isOpen ? (
+      <FlexView onClick={onMenuClick}>
+        {this.props.menuRenderer ?
+          this.props.menuRenderer(options) : <Menu {...{ options }} style={{ maxHeight: height }} />
```

Comment:
I know it was already like this, but this `{...{ options }}` is pretty strange :D

maybe with `options={options}`?